src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -1077,6 +1084,9 @@ def _bigbird_block_rand_mask(
             raise ValueError("Error the number of blocks needs to be same!")
 
         rand_attn = np.zeros((from_seq_length // from_block_size - 2, num_rand_blocks), dtype=np.int32)
+        # During inference (eval) no randomness
+        if deterministic:
+            return rand_attn
```

Comment:
The alternative would be to rename `deterministic` to `training` everywhere in this function (this would be fine in PyTorch, but we prefer `deterministic` in Flax!)
```suggestion
        # During inference (eval) no randomness
        if not training:
            return rand_attn
```

Reply:
Oh yeah, doing stuff while being on the vacation can be a little bit chaotic :)